tests/suite-metrics/TestComputeMetrics.bat

```diff
@@ -0,0 +1,27 @@
+@echo off
+:: nominal check without error
+call ..\..\suite-metrics\ComputeMetrics.bat "c:\Program Files\ANSYS Inc\v242\SCADE" "Model/Model.etp" "MetricRule" false
```

Comment:
ComputeMetrics.bat only takes 3 parameters, right ?
The false is when using the action for the checkout.

Reply:
Fixed in c51ca6d.

Reply:
Fixed in 1c2dd7a.